pkg/manifests/manifests.go

```diff
@@ -40,9 +38,9 @@ const (
 	ServingCertSecretAnnotation = "service.alpha.openshift.io/serving-cert-secret-name"
 
 	// OwningClusterIngressLabel should be applied to any objects "owned by" a
```

Comment:
Due to the name change, I believe `OwningClusterIngressLabel` should be updated to `OwningIngressControllerLabel`.